Simplified/AppInfrastructure/NYPLLibraryNavigationController.m

```diff
@@ -11,70 +22,27 @@
 #import "NYPLSettingsPrimaryTableViewController.h"
 #endif
 
-#import "SimplyE-Swift.h"
 
-#if defined(FEATURE_DRM_CONNECTOR)
-#import <ADEPT/ADEPT.h>
-#endif
+@interface NYPLLibraryNavigationController ()
 
-@implementation NYPLMyBooksNavigationController
+@end
```

Comment:
it is unfortunate Github shows this as a rename. It is not. NYPLLibraryNavigationController is a new superclass for NYPLCatalogNavigationController, NYPLMyBooksNavigationController, NYPLHoldsNavigationController.